src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -178,14 +182,14 @@ def build_actions(
         resolve_button = MessageAction(
             name="status",
             label="Unresolve",
-            value="unresolved",
+            value="unresolved:ongoing",
         )
 
     if status == GroupStatus.IGNORED:
         ignore_button = MessageAction(
             name="status",
-            label="Stop Ignoring",
-            value="unresolved",
+            label="Stop Archive" if has_escalating else "Stop Ignoring",
```

Comment:
Do you have a screenshot of this? I can't picture this in context.

If my intuition is right maybe we should use "Mark as Ongoing". "Stop Archive" does not sound quite right.
```suggestion
            label="Mark as Ongoing" if has_escalating else "Stop Ignoring",
```